setup.py

```diff
@@ -105,6 +107,7 @@ def main():
         "tabulate",
         "tzlocal",
         "ujson",
+        "v4_proto",
```

Comment:
this will generate a conflict when building the wheels of the Hummingbot library. can you remove it from here?

Reply:
@yancong001 did you see this comment?